kubernetes/workflows.yaml

```diff
@@ -114,6 +114,13 @@ workflows:
 
             Also supported are :code:`env` and :code:`volumes` for defining the environment variables and
             volumes specific to this step.
+        - name: generateName
+          description: The prefix for all jobs created by :code:`Honeydipper`, defaults to :code:`honeydipper-job-`.
+        - name: useBerglas
+          description: >
+            Indicates if the environment variables should be parsed by
+            :code:`Berglas` to fetch the secrets. See `Berglas website <https://github.com/GoogleCloudPlatform/berglas>`_ for
```

Comment:
might also call out here that the shell commands would be magically prepended with `berglas exec --` if that's how it works?

Reply:
Updated! Also added a parameter `berglas_files` to create files.